manifold_sampling/py/manifold_sampling_primal.py

```diff
@@ -60,7 +60,7 @@ def manifold_sampling_primal(hfun, Ffun, x0, L, U, nfmax, subprob_switch):
         pass
 
     n, delta, printf, fq_pars, tol, X, F, h, Hash, nf, successful, xkin, Hres = check_inputs_and_initialize(x0, F0, nfmax)
-    flag, x0, __, F0, L, U, xkin = checkinputss(hfun, x0, n, fq_pars["npmax"], nfmax, tol["gtol"], delta, 1, len(F0), F0, xkin, L, U)
+    flag, x0, __, F0, L, U, xkin = checkinputss(hfun, np.atleast_2d(x0), n, fq_pars["npmax"], nfmax, tol["gtol"], delta, 1, len(F0), np.atleast_2d(x0), np.atleast_2d(F0), xkin, L, U)
```

Comment:
@jmlarson1 What is up with this additional x0?

Reply:
1. checkinputss has `x0` and `X_init` arguments. 
2. So far, MS-P doesn't support more than one initial point (so `x0` is `X_init`)
3. I wouldn't worry about MS-P API yet.

Reply:
But isn't this conflating X_0 and Options.X_init?